### ISSUE TITLE
Changing SizesWindow's rootViewController after initial setup doesn't remove the old child view

### ISSUE THREAD

**Issue by frankdilo** [marcosgriselli/Sizes#19]
## Expected Behavior

Only `ViewB` should be on screen.

## Actual Behavior

`ViewA` is on screen on top of `ViewB`.

## Steps to Reproduce the Problem

  1. Set `SizesWindow` as app window in AppDelegate
  1. Set the root view controller of the window to `ViewControllerA`
  1. Later on in the app, change the root view controller of the window to `ViewControllerB`
  1. Observe that both `ViewA` and `ViewB` are on screen

**Comment by frankdilo** [comment]
I already spotted the problem and can provide a PR with a fix:

```swift
open class SizesWindow: UIWindow {
    
    public let sizesViewController = SizesViewController()
    
    override open var rootViewController: UIViewController? {
        get {
            return sizesViewController.containedController
        }
        set {
            guard let root = newValue, !root.isKind(of: SizesViewController.self) else {
                return
            }
            super.rootViewController = sizesViewController
            sizesViewController.contain(viewController: root)
        }
    }
}
```

In the setter, before adding another child view controller we should remove the existing one if it's there. I can submit a PR.

**Comment by marcosgriselli** [comment]
Exactly. I think we need to modify the `contain(viewController: _)` method to remove the previously contained view controller. I'd really appreciate a PR :).

**Comment by frankdilo** [comment]
Done! 🙌

**Comment by marcosgriselli** [comment]
Closed in #20
